packages/open-flow/src/workbench/browser/runtime/locales/fr.json

```
@@ -0,0 +1,496 @@
+  "sidebar": {
+    "createFirstFlow": "Créer un Flow",
+    "createResource": "Créer un {{kind}}",
+    "deleteFlow": "Supprimer {{name}}",
+    "deleteFlowConfirm": "Supprimer “{{name}}” ?",
+    "deleteFlowLiveNote": "Son déploiement en ligne et ses liaisons de Trigger seront également retirés.",
+    "draft": "Brouillon",
+    "flowActions": "Actions pour {{name}}",
+    "flowLiveOnly": "Ce Flow n'a qu'une révision en ligne.",
+    "flows": "Flows",
+    "liveAndDraft": "En ligne + Brouillon",
+    "liveOnly": "En ligne uniquement",
```

Biome (2.5.7)

[error] The key liveOnly was already declared.

(lint/suspicious/noDuplicateObjectKeys)